helm/hpcc/templates/_warnings.tpl

```diff
@@ -52,4 +52,78 @@ Pass in dict with root and warnings
   {{- $_ := set $warning "msg" (printf "The configuration contains ephemeral planes: %s" (print (uniq $match.ephemeral)) )  -}}
   {{- $_ := set $ctx "warnings" (append $ctx.warnings $warning) -}}
  {{- end -}}
+ {{- /* Warn if resources not defined for any cost component or default cpuRate being used */ -}}
+ {{- if eq .root.Values.global.cost.perCpu 0.0565000000001 -}}
+  {{- $_ := set $ctx "usingDefaultCpuCost" "true" -}}
+ {{- end -}}
+ {{- $_ := set $ctx "missingResources" list -}}
+ {{- $_ := set $ctx "defaultCpuRateComponents" list -}}
+ {{- range $component := .root.Values.eclagent -}}
+  {{- if and (not $component.disabled) (eq $component.type "hthor" ) -}}
```

Comment:
Why the check on $component.type?  I think the alternative value is "roxie" which should also check resources.  Is that for a follow on PR?

Reply:
The reason was that the "roxie" type isn't used in the cost calculation at the moment.  However, I'm going to remove this check as it's unnecessary complication.